src/pages/surveys/[id]/questions/[currentOrder].page.tsx

```diff
@@ -90,9 +140,23 @@ const Question: NextPage = () => {
               />
             </button>
           ) : (
-            <Button label="Submit" className="px-8" />
+            <Button
+              label="Submit"
+              className="px-8"
+              onClick={onSubmitSurvey}
+              disabled={isEmptySurveyResponse || isSurveySubmit}
+            />
           )}
         </div>
+        {surveySubmitError.length !== 0 && (
```

Comment:
```suggestion
        {submitErrorMessage && (
```
As strings aggregates to `false` when empty, the suggestion bellow makes this line easier to understand too :)

Reply:
Updated in [3d184a9](https://github.com/hanam1ni/react-surveyor/pull/89/commits/3d184a9c26fcb80a4da2c9876b79e9f6d350c8ba)